### PR TITLE
docs: distributed Coordinator

### PR DESCRIPTION
**Pull request by thomasten**
Initial docs for the distributed Coordinator.
Not sure if there are more things to document about this? Let me know.

### CODE REVIEW

File: docs/docs/features/recovery.md [hunk 1]
```diff
@@ -10,23 +10,42 @@ In sum, this means that the usual SGX programming model isn't suited for virtual
...
+In this case, the state is encrypted with the SGX seal key and stored on disk.
+When being pinned to a single host, the Coordinator can unseal its state automatically.
+However, when the Coordinator is moved to another physical host, a [manual step](#recovery) is required to ensure the Coordinator's state can be recovered.
+
+### Distributed Coordinator
+
+<enterpriseBanner/>
+
+When you use MarbleRun [with Kubernetes](../deployment/kubernetes.md), you can scale the Coordinator to multiple instances.
+The instances share a common state, which is encrypted and stored as a Kubernetes secret.
```
Comment (m1ghtym0) [inline docs/docs/features/recovery.md:27]:
```suggestion
The instances share a common state, encrypted and stored as a Kubernetes secret.
```

File: docs/docs/features/recovery.md [hunk 2]
```diff
@@ -10,23 +10,42 @@ In sum, this means that the usual SGX programming model isn't suited for virtual
...
 Thus, Marbles can be distributed and rescheduled on arbitrary machines.
+This narrows the challenge of persistent storage down to the Coordinator itself.
 
-Still, the Coordinator itself must keep its state persistent somehow. When being pinned to a single host the default SGX sealing methods are used. However, when the Coordinator is moved to another physical host, a manual step is required to ensure the Coordinator's state can be recovered.
-Therefore, the manifest allows for specifying a designated *Recovery Key*. The Recovery Key is a public RSA key. Upon startup, the Coordinator encrypts its symmetric state-encryption key with this public key. The holder of the corresponding private key can recover the Coordinator, as is described [in the recovery workflow](../workflows/recover-coordinator.md).
+### Single Coordinator
+
+The straightforward way to run MarbleRun is with a single Coordinator.
+In this case, the state is encrypted with the SGX seal key and stored on disk.
+When being pinned to a single host, the Coordinator can unseal its state automatically.
```
Comment (m1ghtym0) [inline docs/docs/features/recovery.md:19]:
```suggestion
When pinned to a single host, the Coordinator can unseal its state automatically.
```

File: docs/docs/features/recovery.md [hunk 3]
```diff
@@ -10,23 +10,42 @@ In sum, this means that the usual SGX programming model isn't suited for virtual
...
+However, when the Coordinator is moved to another physical host, a [manual step](#recovery) is required to ensure the Coordinator's state can be recovered.
```
Comment (m1ghtym0) [inline docs/docs/features/recovery.md:20]:
```suggestion
However, when the Coordinator is moved to another physical host, a [manual step](#recovery) is required to recover the Coordinator's state.
```

File: docs/docs/workflows/recover-coordinator.md [hunk 4]
```diff
@@ -3,6 +3,12 @@
 Different situations require the [recovery](../features/recovery.md) of the Coordinator.
 If the Coordinator fails to unseal its state, it will enter recovery mode.
 
+:::tip
+
+Use [distributed Coordinator instances](../features/recovery.md#distributed-coordinator) to minimize situations where manual recovery is required.
```
Comment (m1ghtym0) [inline docs/docs/workflows/recover-coordinator.md:8]: I'm unsure if "minimize situations" is the proper term here.
Some suggestions:
* reduce the frequency
* diminish the likelihood
* lessen the occurrences

File: docs/docs/getting-started/coordinator.md [hunk 5]
```diff
@@ -9,6 +9,8 @@ The Coordinator can be configured with several environment variables:
 * `EDG_COORDINATOR_DNS_NAMES`: The DNS names for the cluster's root certificate
 * `EDG_COORDINATOR_SEAL_DIR`: The file path for storing sealed data
 
+When you use MarbleRun [with Kubernetes](../deployment/kubernetes.md), you can [scale the Coordinator to multiple instances](../features/recovery.md#distributed-coordinator) to increase availability and minimize situations where [manual recovery](../workflows/recover-coordinator.md) is required.
```
Comment (m1ghtym0) [inline docs/docs/getting-started/coordinator.md:12]: Same here with "minimize situations", see below

**Review by m1ghtym0 — COMMENTED** [review]
Thanks, Thomas! Content-wise, I have nothing to add.
Just some minor language annotations.

**Review by m1ghtym0 — APPROVED** [review]

**Review by daniel-weisse — APPROVED** [review]

**Pull request merged**